ocs_ci/ocs/resources/pod.py

```diff
@@ -158,6 +159,30 @@ def exec_cmd_on_pod(
             rsh_cmd, out_yaml_format, secrets=secrets, timeout=timeout, **kwargs
         )
 
+    def exec_s3_cmd_on_pod(self, command, mcg_obj):
```

Comment:
`craft_s3_command` was designed to not necessarily require `mcg_obj`, and can work without it.
Change the function to allow usage without `mcg_obj`, and with the other parameters `craft_s3_command` receives